src/Uno.Templates/content/unoapp/.template.config/template.json

```diff
@@ -1234,6 +1253,21 @@
       "dataType": "bool",
       "value": "(httpEvaluator && useConfiguration)"
     },
+    "useHttpDefault": {
+      "type": "computed",
+      "datatype": "bool",
+      "value": "useHttp && http == 'default'"
+    },
```

Comment:
This would need to change to take over the `useHttp` computed parameter above. We should have `useHttp`, `useHttpRefit`, and `useHttpKiota`. Where it is implicit that `useHttp` is just using the `default` option. 

And there should be further edits since certain parts of the template are relying on `useHttp` to include certain code but those need to be changed to rely on `useHttpRefit` instead. Since things like the WeatherForecast service code in the Recommended template uses Refit